src/plone/restapi/services/content/add.py

```diff
@@ -88,3 +111,27 @@ def reply(self):
         serialized_obj["@id"] = obj.absolute_url()
 
         return serialized_obj
+
+    def get_object(self, key):
+        portal = getMultiAdapter(
+            (self.context, self.request), name="plone_portal_state"
+        ).portal()
+        catalog = getToolByName(self.context, "portal_catalog")
+
+        if isinstance(key, six.string_types):
```

Comment:
We are trying to get rid of this `six.string_types` check in https://github.com/plone/plone.restapi/pull/876 (sorry I need to review @ale-rt's comment in there).

Reply:
@erral it is not possible to use the CMFPlone function that does this because plone.restapi needs to work with other Plone versions as well.

Reply:
@erral sorry, got this wrong. Ignore my comment!

Reply:
@erral mmm I forgot this, @tisto maybe, do we want to convert this get_object into a reusable utils function?

I can refactor it and include #876 updates. However, I don't get if @ale-rt 's comment is something that we want to improve.